_search-plugins/search-relevance/stats-api.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Search relevance Stats API
+nav_order: 65
+parent: Search relevance
+has_children: false
+---
+
+# Search Relevance Stats API
+Introduced 2.7
+{: .label .label-purple }
+
+The Search Relevance Stats API provides information about the operations of the [Search Relevance plugin](https://github.com/opensearch-project/dashboards-search-relevance). The Search Relevance plugin processes operations sent by the [Compare Search Results]({{site.url}}{{site.baseurl}}/search-plugins/search-relevance) Dashboards tool.
```

Comment:
Are we capitalizing "Compare Search Results" whenever we refer to that Dashboard's functionality?

Reply:
Yes